### PR TITLE
Style: Adds missing styling

### PR DESCRIPTION
**Pull request by FritzHoing**
- Adds CSS
- minor UI adjustments

klips-style

### CODE REVIEW

File: easy-to-use-api/klips-url-generator/src/App.tsx [hunk 1]
```diff
@@ -108,50 +111,50 @@ const App: React.FC = () => {
...
       <MapContext.Provider value={map}>
-        <div className='output-wrapper'>
-          <div className='header'>
-            <img src="https://www.klips-projekt.de/wp-content/uploads/2021/02/SAG_KLIPS-Logo_Jan21.png" alt="KLIPS Logo"></img>
-            <h2>Widget URL Generator</h2>
+        <ConfigProvider 
+        >
+          <div className='output-wrapper'>
+            <div className='header'>
+              <img className='logo' src="https://www.klips-projekt.de/wp-content/uploads/2021/02/SAG_KLIPS-Logo_Jan21.png" alt="KLIPS Logo"></img>
```
Comment (hblitza) [inline easy-to-use-api/klips-url-generator/src/App.tsx:138]: just reminder, we should host this on our server

File: easy-to-use-api/klips-url-generator/src/style.css [hunk 2]
```diff
@@ -10,110 +11,91 @@ body,
...
+  flex-direction: row;
 }
 
-.nominatim-search {
-  width: 100%;
+.personal-title-selector {
+  display: flex;
+  flex-direction: column;
+  padding: 30px;
 }
```
Comment (hblitza) [inline easy-to-use-api/klips-url-generator/src/style.css:101]:
```suggestion
}

```

**Comment by hblitza** [comment]
nice work @FritzHoing lgtm. the selections in "band" is in English, we should harmonize this in German i think

**Review by hblitza — APPROVED** [review]

File: easy-to-use-api/klips-url-generator/src/App.tsx (hunk 1 above)
Comment (FritzHoing) [reply easy-to-use-api/klips-url-generator/src/App.tsx:138]: Good point!

**Comment by FritzHoing** [comment]
> nice work @FritzHoing lgtm. the selections in "band" is in English, we should harmonize this in German i think

Yes we should definitely do that!

**Pull request merged**